src/ir/constant-int.cc

```diff
@@ -74,9 +82,18 @@ NAN_METHOD(ConstantIntWrapper::getFalse) {
     info.GetReturnValue().Set(ConstantIntWrapper::of(constant));
 }
 
+NAN_METHOD(ConstantIntWrapper::getValueAsString) {
```

Comment:
What about we rename the method to toString()? In this case there is no need to ever deprecate this method. What do you think? I'm not sure if it is a good idea.